src/gui/csvImport/CsvImportWidget.cpp

```diff
@@ -0,0 +1,289 @@
+﻿/*
+ *  Copyright (C) 2016 Enrico Mariotti <enricomariotti@yahoo.it>
+ *
+ *  This program is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 2 or (at your option)
+ *  version 3 of the License.
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include "CsvImportWidget.h"
+#include "ui_CsvImportWidget.h"
+#include "gui/MessageBox.h"
+
+#include <QFile>
+#include <QFileInfo>
```

Comment:
@seatedscribe I think the general guideline for imports in the project is:
1. _ui and associated .h file
2. QT imports
3. imports from the project.
These three sections separated by a blank line.

Look at `SearchWidget.cpp`'s import section, for example.